bn-apps/memberships-management/membership-service/src/main/kotlin/net/corda/businessnetworks/membership/member/GetMembershipsFlow.kt

```diff
@@ -42,35 +40,32 @@ data class MembershipsListResponse(val memberships: List<StateAndRef<MembershipS
  */
 @InitiatingFlow
 @StartableByRPC
-class GetMembershipsFlow(private val forceRefresh: Boolean = false, private val filterOutInactive : Boolean = true) : FlowLogic<Map<Party, StateAndRef<MembershipState<Any>>>>() {
+class GetMembershipsFlow(bno : Party, private val forceRefresh: Boolean = false, private val filterOutInactive : Boolean = true) : BusinessNetworkAwareInitiatingFlow<Map<Party, StateAndRef<MembershipState<Any>>>>(bno) {
```

Comment:
bno parameter added but not used anywhere?
If it is intended to be used at a later date it should be mentioned somewhere.
Also, is it possible to ensure that the bno parameter is BNO, if so it should be done.

Reply:
bno is actually used in the code, please check. BNO verification logic is done inside `BusinessNetworkAwareInitiatingFlow ` as per commend above